spec/features/log_out_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'peeper'
```

Comment:
Is this test really testing the log out feature? I would expect there to be some set up where a user is logged in first, then logged out?